lrucache.go

```diff
@@ -221,7 +226,14 @@ func (c *LRUCache) purge() {
 			if len(group.nodes) == 0 {
 				delete(c.groups, group.key)
 			}
+			fmt.Printf("Evicted node '%s' - group '%s'\n", node.key, group.key)
```

Comment:
Are we ok with using fmt.Printf instead of logs (even though both should write to stdout, if so - should it be JSON formatted)?

Reply:
Yeah right, let me change it to JSON format